### PR TITLE
suggested changes for documentation

### PR DESCRIPTION
**Pull request by Legendofire**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -110,9 +110,9 @@ const { model, fields, onSubmit } = useForm<{ name: string; phone: number }>({
 
 The validations object will have the same keys as your model object. Each key can take one of the following values:
 
-- A string that contains prebuild usual validations like `required, email, number, json, website, etc.`
+- A string that contains prebuild usual validations, currently we support `required, email, number, json, website`
```
Comment (Legendofire) [inline Readme.md:113]: since we have a finite list here the `etc.` can be deceiving.

File: Readme.md [hunk 2]
```diff
@@ -192,18 +194,18 @@ There are several properties exposed from the hook to deal with the state throug
 | error            | any error thrown within the handleSubmit function is stored here as an error object                    |
 | submissionStatus | Displays the current status of the submission process                                                  |
 
-#### Submission Status
+### Submission Status
```
Comment (Legendofire) [inline Readme.md:197]: quickfix to have some space between the title and the table above it. if we want to stick to h4 then we can change the css file (not sure if it's also generated)

File: Readme.md [hunk 3]
```diff
@@ -192,18 +194,18 @@ There are several properties exposed from the hook to deal with the state throug
...
 
 This variable contains the current state of the form submission process.
 
 | Status     | Details                                                                     |
 | ---------- | --------------------------------------------------------------------------- |
-| idle       | The form has not yet been submitted. It also applied when the form is reset |
+| pristine   | The form has not yet been submitted. It also applied when the form is reset |
```
Comment (Legendofire) [inline Readme.md:203]: For me Idle suggests something happening in the background, and idle means it's not happening anymore. i like Angular's wording here more `Pristine` means untouched

Comment (leoggonzalez) [reply Readme.md:203]: I think idle is a synonym of "stand by" it means waiting to do something. The flow is:

idle -> submitting -> submitted / error -> reset -> idle

https://www.dictionary.com/browse/idle

**Comment by leoggonzalez** [comment]
I'll push the changes directly to master

**Pull request closed without merging**